packages/fields/src/components/es-multi-checkbox/es-multi-checkbox.tsx

```diff
@@ -37,8 +37,8 @@ export class EsMultiCheckbox {
     render() {
         return (
             <div class={{ field: true, invalid: !!this.invalid }}>
-                {this.options.map(option => (
-                    <label>
+                {this.options.map((option, i) => (
+                    <label key={option.value} tabIndex={i}>
```

Comment:
`tabindex` should be `0`, also needs to be `undefined` if the field is disabled.